frontend/openchat-client/src/openchat.ts

```diff
@@ -4995,6 +4997,31 @@ export class OpenChat extends OpenChatAgentWorker {
         });
     }
 
+    submitProposal(governanceCanisterId: string, proposal: CandidateProposal): Promise<boolean> {
+        return this.sendRequest({
+            kind: "submitProposal",
+            governanceCanisterId: Principal.fromText(governanceCanisterId),
+            proposal,
+        }).then((resp) => {
+            if (resp.kind === "success" || resp.kind === "retrying") {
+                return true;
+            }
+
+            let error = resp.kind;
+            if ("error" in resp) {
+                error += " " + resp?.error;
+            }
+            console.debug("Failed to submit proposal", error);
```

Comment:
I think these `console.debug` lines are redundant as the logger will already log to console as well.